internal/configuration/settings/portforward.go

```diff
@@ -162,6 +175,9 @@ func (p *PortForwarding) read(r *reader.Reader) (err error) {
 			"PRIVATE_INTERNET_ACCESS_VPN_PORT_FORWARDING_STATUS_FILE",
 		))
 
+	p.Command = r.Get("VPN_PORT_FORWARDING_UP_COMMAND",
```

Comment:
Also please add `VPN_PORT_FORWARDING_UP_COMMAND=` to the Dockerfile's ENV list thanks!